configure.ac

```diff
@@ -2411,6 +2411,67 @@ then
 fi
 # }}}
 
+# --with-libdpdk {{{
+AC_ARG_WITH(libdpdk, [AS_HELP_STRING([--with-libdpdk@<:@=PREFIX@:>@], [Path to the DPDK build directory.])],
+[
+	if test "x$withval" != "xno" && test "x$withval" != "xyes"
+	then
+		with_dpdk_path="$withval"
+		with_libdpdk="yes"
+	else
+		if test "x$withval" = "xno"
+		then
+			with_libdpdk="no (disabled)"
+		fi
+	fi
+], [with_libdpdk="no"])
+
+if test "x$with_libdpdk" = "xyes"
+then
+	RTE_BUILD="$with_dpdk_path"
+	DPDK_INCLUDE="$RTE_BUILD/include"
+	AC_CHECK_FILE([$DPDK_INCLUDE/rte_config.h], [LOCAL_DPDK_INSTALL=true],
+		[AC_CHECK_FILE([$DPDK_INCLUDE/dpdk/rte_config.h],
+		[DPDK_INCLUDE=$DPDK_INCLUDE/dpdk], [])])
+	DPDK_LIB_DIR="$RTE_BUILD/lib"
+	DPDK_EXTRA_LIB=""
+
+	SAVE_CFLAGS="$CFLAGS"
+	SAVE_LDFLAGS="$LDFLAGS"
+	CFLAGS="$CFLAGS -I$DPDK_INCLUDE"
+	if test "x$LOCAL_DPDK_INSTALL" != "xtrue"
```

Comment:
We treat them differently because it's common to install DPDK locally. For a local install headers are in $RTE_BUILD/include. If the install is to /usr/lib, they are in /usr/include/dpdk. 

In my next commit I will change all "true" and "false" values to "yes" and "no".